packages/shared/styles/components/SfGallery.scss

```
@@ -121,3 +121,14 @@ $gallery__stage-cursor: zoom-in !default;
     }
   }
 }
+
+.sf-gallery__zoom {
+  z-index: 1000;
```

```suggestion
  z-index: 1;
```
I think it's too big :)

---

yes yes it's too big :) srry fixed!

---

thanks! :)